cmd/karavictl/cmd/root.go

```diff
@@ -53,6 +63,7 @@ func NewRootCmd() *cobra.Command {
 	}
 	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.karavictl.yaml)")
 	rootCmd.Flags().BoolP("toggle", "t", false, "Help message for toggle")
+	rootCmd.PersistentFlags().StringP("admin_token", "f", "", "Specify the admin token file")
```

Comment:
Nitpicking, but `admin-token` instead of `admin_token` will follow the pattern of other flags; like `--access-token-expiration`.